BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBUserProfile.m

```diff
@@ -31,15 +31,29 @@
 //
 
 #import "_SBBUserProfile.h"
+#import "ModelObjectInternal.h"
 #import "NSDate+SBBAdditions.h"
 
+@import ObjectiveC;
+
 @interface _SBBUserProfile()
 
 @end
 
+// see xcdoc://?url=developer.apple.com/library/etc/redirect/xcode/ios/602958/documentation/Cocoa/Conceptual/CoreData/Articles/cdAccessorMethods.html
+@interface NSManagedObject (UserProfile)
+
+@property (nullable, nonatomic, retain) NSString* email;
+
+@property (nullable, nonatomic, retain) NSString* firstName;
+
+@property (nullable, nonatomic, retain) NSString* lastName;
+
+@end
+
 @implementation _SBBUserProfile
```

Comment:
technically, yes, it's cached automatically by SBBObjectManager when it creates the SBBUserProfile object from Bridge JSON--for all the good it does, since I haven't updated the interfaces to SBBUserManager to make use of the cache. I'll do that.